src/widgetastic/widget/input.py

```diff
@@ -60,7 +60,7 @@ def fill(self, value):
             return False
         # Clear and type everything
         self.browser.click(self)
-        self.browser.clear(self)
+        self.browser.execute_script("arguments[0].value = '';", self)
```

Comment:
It should be done in `browser.clear` method as a case for chrome and a specific version.

Reply:
If it's specific to a browser or some location better to go with a local widget (overwrite fill method).
As its core lib; I think if we fallow the selenium default call (execute `clearElement` cmd) will be a good option.

Reply:
done